src/test/java/com/hocs/test/pages/to/Triage.java

```diff
@@ -18,9 +18,9 @@ public void selectToChangeTheBusinessArea() {
         waitForPageWithTitle("Transfer To Business Area");
     }
 
-    public void selectADifferentBusinessArea() {
-        String newBusinessArea = selectDifferentRadioButtonFromGroupWithHeading("Business Area");
-        setSessionVariable("businessArea").to(newBusinessArea);
+    public void selectADifferentBusinessArea(String businessArea) {
+        selectSpecificRadioButtonFromGroupWithHeading(businessArea, "Business Area");
```

Comment:
We can probably do away with this and instead just use the selectABusinessArea method from Registration stage